package.json

```diff
@@ -25,9 +25,10 @@
   "dependencies": {
     "async": "2.6.1",
     "deasync": "0.1.12",
+    "jquery": "3.3.1",
```

Comment:
Scripting docs [indicate](https://zapier.com/developer/documentation/v2/scripting/#available-libraries) that WB has access to jquery `1.8.3`. Is there concern around changing this?

Reply:
Looks like the doc is out of date. We're actually using [2.1.4](https://github.com/zapier/lambda-wrapper/blob/6bb8bfbdc1268fe2167ab9f67cb4e61c135673f3/package.json#L19). Honestly, I'm not worrying too much about inconsistent dep versions, because the other libraries are already different. For example, we're using lodash here while WB uses underscore. That said, I've pinned jquery to the latest compatible version 2.2.4 in 7464768, just to be safe.